docs/reference/status.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Network status
-description: Get the status of the ConsenSys zkEVM network
+description: Get the status of the Linea network
 sidebar_position: 1
 ---
 
 # Get the network status
 
-To find the status of ConsenSys zkEVM network, you'll need to:
+To find the status of the Linea network, you'll need to:
 
 1. Navigate to the [Infura status page](https://status.infura.io/).
```

Comment:
Is this still accurate that users need to check the Infura page for the network status?

Reply:
yep